services/brig/src/Brig/User/API/Search.hs

```diff
@@ -132,30 +134,36 @@ routesInternal = do
 
 -- Handlers
 
+-- TODO question: would it make sense to create a second endpoint with a mandatory domain?
+-- That would make it easier to see from logs whether the old endpoint is still in use or can be removed.
+--
+-- TODO question: would it make sense to augment the 'SearchResult'
+-- to include a separate entry for exact handle match? This may allow clients to handle that differently.
+-- Not sure if useful or not - perhaps discuss with clients?
```

Comment:
Seems like there is already demand for this: https://wearezeta.atlassian.net/browse/BE-262